travis.sh

```diff
@@ -170,5 +170,8 @@ if [ $NUMNOTVERIFIED -ne 2 ] || [ $NUMNOMAGIC -ne 0 ]; then
 	exit -1;
 fi
 
+mysql domjudge -e 'SELECT max(eventid) FROM event'
+mysql domjudge -e 'SELECT * FROM event WHERE eventid=112\G'
```

Comment:
what is \G here? a typo?

Reply:
No, \G prints SQL rows per line instead of column.